### PR TITLE
[smtk-import] Move to current mdbtools github repo

### PR DESCRIPTION
**Pull request by salvadorcunat**
Original mdbtools developer gave up the project some time ago, and it's continued with his permission in a forked repo:
	https://github.com/mdbtools/mdbtools.git

There was a nasty bug in libmdb, triggered under some rare circustances, that is solved in the new repo which is, BTW, under current development.

Move our scripts to the new repo and set our working version to the latest release tag, currently, "v1.0.0"

### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [X] Build system change
- [ ] Documentation change
- [ ] Language translation

### Mentions:

@dirkhh @atdotde

### CODE REVIEW

**Review by mikeller — APPROVED** [review]
Looks good to me.

File: scripts/smtk2ssrf-build.sh [hunk 1]
```diff
@@ -108,15 +108,16 @@ else
...
 			aborting "Problem downloading/updating mdbtools"
 	fi
 
 	echo "----> Building mdbtools ..."
 	echo "----> This will display a lot of errors and warnings"
 	cd "$BASEDIR"/mdbtools || aborting "Couldn't cd into $BASEDIR/mdbtools"
+	git checkout "v1.0.0"
 	autoreconf -i -f
-	./configure --prefix "$INSTALL_ROOT" --disable-man --disable-gmdb2 >/dev/null
+	./configure --prefix "$INSTALL_ROOT" --disable-man --with-bash-completion-dir="$INSTALL_ROOT"/share/bash-completion/completions >/dev/null
```
Comment (dirkhh) [inline scripts/smtk2ssrf-build.sh]: that seems just weird. Why would we set a bash completion dir?

**Review by dirkhh — COMMENTED** [review]
The build failures for Windows/Mac/Android make perfect sense to me - that's something that I need to fix in the workflows. I can't allow a pull request to upload binaries.
I am a bit confused by the failure of the Qt5.12 test build - but then again, maintaining that one is kinda silly and something I want to revisit as well.

So the main reason not to merge this right now is my comment about the bash completions...

File: scripts/smtk2ssrf-build.sh (hunk 1 above)
Comment (salvadorcunat) [reply scripts/smtk2ssrf-build.sh]: Some time ago, this version of mdbtools refused to build unless this was explicitly set.  Probably a debian artifact.  Now it builds just fine without it, so I just force pushed the modification to the PR removing the bash-completion thing.

Comment (salvadorcunat) [reply scripts/smtk2ssrf-build.sh]: Sorry, new force push.  The bash completion thing is set in configure.ac in mdbtools 
and must be explicitly turned off.

**Comment by mikeller** [comment]
@salvadorcunat: Apologies, but I lost your latest changes when I rebased this to test the latest changes to our CI/CD setup - can you please push them again?

**Comment by salvadorcunat** [comment]
Hi @mikeller .  It's pushed again.

**Review by mikeller — APPROVED** [review]

**Pull request merged**